### PR TITLE
feat: IDP-213 enabled retry to APIM for PDV requests

### PR DESCRIPTION
**Pull request by marco-di-lisa**
### List of changes

Added retry to APIM product policy to address 429 errors from PDV

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

Must be merged after PR https://github.com/pagopa/cstar-infrastructure/pull/840

### CODE REVIEW

**Review by francescopersico — CHANGES_REQUESTED** [review]

File: src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml.tpl [hunk 1]
```diff
@@ -728,6 +782,38 @@ components:
...
+    InitiativesStatusDTO:
+      type: object
+      required:
+        - initiativeId
+        - status
+      properties:
+        initiativeId:
+          type: string
+        status:
+          type: string
```
[the remark this answers is not in the document]
Comment (marco-di-lisa) [reply src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml.tpl:814]: @francescopersico fixed in commit [ad75b53](https://github.com/pagopa/cstar-infrastructure/pull/832/commits/ad75b534b5b0113a55747142dada31a8ce236d28)

File: src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml [hunk 2]
```diff
@@ -728,6 +782,43 @@ components:
...
+          type: string
+        idInstrument:
+          type: string
+        status:
+          enum:
+            - ACTIVE
+            - INACTIVE
+            - PENDING_ENROLLMENT_REQUEST
+            - PENDING_DEACTIVATION_REQUEST
+            - ENROLLMENT_FAILED
```
Comment (forrest57) [inline src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml:821]: please add a `type` and `description` field to the enum, else the generator will fail on `ErrorDTO`

Comment (marco-di-lisa) [reply src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml:825]: @forrest57 fixed in [008170a](https://github.com/pagopa/cstar-infrastructure/pull/832/commits/008170a48e1c5e40156552d9c22fa44b1cc09def)

**Comment by marco-di-lisa** [comment]
@francescopersico removed accrued field from refunds in commit [1a1f794](https://github.com/pagopa/cstar-infrastructure/pull/832/commits/1a1f7948dd0baea5a6c76243e1e305208219fdad)

File: src/domains/idpay-app/99_variables.tf [hunk 3]
```diff
@@ -184,6 +184,30 @@ variable "pdv_timeout_sec" {
   default     = 15
 }
 
+variable "pdv_retry_count" {
```
Comment (GiovanniMancini) [inline src/domains/idpay-app/99_variables.tf:187]: maybe all variables to config pdv retries can be put in the same variable

**Pull request merged**